keystore/src/entities/platform/generic/mls/epoch_encryption_keypair.rs

```diff
@@ -201,8 +159,8 @@ impl EntityBase for MlsEpochEncryptionKeyPair {
         let mut updated = 0;
         for id in ids {
             updated += transaction.execute(
```

Comment:
Nitpick: I'd rename this to `deleted`.

Reply:
I'd like to deal with this when eliminating code duplication, since this would mean that we touch every file where the Entity trait is implemented.